packages/flex-dev-utils/src/packages/lib/getLatestFlexUIVersion.ts

```diff
@@ -0,0 +1,32 @@
+import semver from 'semver';
+
+import getRegistryVersion from './getRegistryVersion';
+
+const flexUI = '@twilio/flex-ui';
+
+/**
+ * Returns the latest flex ui version for a given major
+ * Will search through latest, beta, and alpha versions for a matching verion (in that order)
+ * @param version the flex ui major version
+ */
+export default async function getLatestFlexUIVersion(version: number): Promise<string> {
+  const latest = await getRegistryVersion(flexUI, 'latest');
+  const latestMajor = latest ? semver.coerce(latest.version as string)?.major : undefined;
+  if (latestMajor && latestMajor >= version) {
```

Comment:
Shouldn't the comparision always be `===`? Like, if you ask for version 2 but the latest is 3, this should _not_ return it.